qa/workunits/rados/test_rados_tool.sh

```diff
@@ -559,13 +559,312 @@ function test_put()
   rm -rf ./rados_object_10k ./rados_object_10k_out ./rados_object_offset_0_out ./rados_object_offset_1000_out
 }
 
+function test_stat()
+{
+  # create file in local fs
+  dd if=/dev/urandom of=rados_object_128k bs=64K count=2
+
+  # rados df test (replicated_pool):
+  $RADOS_TOOL purge $POOL --yes-i-really-really-mean-it
+  $CEPH_TOOL osd pool rm $POOL $POOL --yes-i-really-really-mean-it
+  $CEPH_TOOL osd pool create $POOL 8
+
+
+  # put object with 1 MB gap in front
+  $RADOS_TOOL -p $POOL put $OBJ ./rados_object_128k --offset=1048576
+  MATCH_CNT=0
+  for i in {1..60}
+  do
+    IN=$($RADOS_TOOL -p $POOL df | grep $POOL ; [[ ! -z $? ]] && echo "")
+    [[ -z $IN ]] && sleep 1 && continue
+    IFS=' ' read -ra VALS <<< "$IN"
+
+    # verification is a bit tricky due to stats report's eventual model
+    # VALS[1] - USED
+    # VALS[2] - USED units
+    # VALS[3] - OBJECTS
+    # VALS[5] - COPIES
+    # VALS[12] - WR_OPS
+    # VALS[13] - WR
+    # VALS[14] - WR uints
+    # implies replication factor 3
+    if [ ${VALS[1]} == 384 ] && [ ${VALS[2]} == "KiB" ] && [ ${VALS[3]} == "1" ] && [ ${VALS[5]} == "3" ] && [ ${VALS[12]} == "1" ] && [ ${VALS[13]} == 128 ] && [ ${VALS[14]} == "KiB" ]
```

Comment:
http://pulpito.ceph.com/kchai-2018-11-15_09:38:15-rados-wip-kefu-testing-2018-11-15-1539-distro-basic-smithi/3257828/

```
2018-11-15T10:34:32.456 INFO:tasks.workunit.client.0.smithi181.stderr:+ IN='trs_pool  256 KiB       1      0      2                  0       0        0      0 0 B      1 128 KiB        0 B         0 B '
2018-11-15T10:34:32.456 INFO:tasks.workunit.client.0.smithi181.stderr:+ [[ -z trs_pool  256 KiB       1      0      2                  0       0        0      0 0 B      1 128 KiB        0 B         0 B  ]]
2018-11-15T10:34:32.456 INFO:tasks.workunit.client.0.smithi181.stderr:+ IFS=' '
2018-11-15T10:34:32.456 INFO:tasks.workunit.client.0.smithi181.stderr:+ read -ra VALS
2018-11-15T10:34:32.456 INFO:tasks.workunit.client.0.smithi181.stderr:+ '[' 256 == 384 ']'
2018-11-15T10:34:32.457 INFO:tasks.workunit.client.0.smithi181.stderr:+ MATCH_CNT=0
```
seems "COPIES" is always 2.

Reply:
```
2018-11-28T01:33:02.414 INFO:tasks.workunit.client.0.smithi028.stderr:+ [[ -z trs_pool  1.1 MiB       1      0      3                  0       0        0      0 0 B      1 128 KiB        0 B         0 B  ]]
2018-11-28T01:33:02.414 INFO:tasks.workunit.client.0.smithi028.stderr:+ '[' 1.1 '!=' 384 ']'
2018-11-28T01:33:02.414 INFO:tasks.workunit.client.0.smithi028.stderr:+ die 'Failed to retrieve proper pool stats within 60 seconds'
2018-11-28T01:33:02.415 INFO:tasks.workunit.client.0.smithi028.stderr:+ echo 'Failed to retrieve proper pool stats within 60 seconds'
2018-11-28T01:33:02.415 INFO:tasks.workunit.client.0.smithi028.stderr:+ exit 1
```

http://pulpito.ceph.com/kchai-2018-11-28_00:59:04-rados-wip-kefu2-testing-2018-11-27-2345-distro-basic-smithi/3286644/